pyiron_base/job/core.py

```diff
@@ -655,7 +655,7 @@ def _list_all(self):
         h5_dict["groups"] += self._list_ext_childs()
         return h5_dict
 
-    def copy(self):
+    def copy_self(self):
```

Comment:
```suggestion
    def copy(self):
```
Imo not necessary.

Reply:
I think it's to make sure the right copy gets called where there's multiple inheritance

Reply:
As Liam said. I could not isolate which copy it was calling in the first place, but it was definitely not calling the one defined in the class.

Reply:
Ah, that's big stinky on our part then.

Reply:
I've traced it in a debugger and the issue is that `GenericJob.copy()` doesn't call `super()`.  We should fix that in separate issue.  Apparently this caused a recursion error in some master job before.

Reply:
Aha! And this is the same `RecursionError` that I try to get around by adding `exception = False` in `pyiron_base/master/generic.py` 

My solution then also requires me to add this exception `except (ValueError, TypeError):`. `__getitem__` throws a TypeError if the item is of NoneType, because I set `child_ids` to `None`.

Reply:
😬

Reply:
@liamhuber someday, sometime in the future I will owe someone a beer. :P

In all seriousness, any suggestions to fix this without using what I consider my 'workaround'? I also do not like the fact that it may cause another Raynol in the future to spend time on this.

Reply:
You can replace `super().some_method(foo)` with `ParentClass.some_method(self, foo)`, provided you can find all the right places to do it.

I think the root of the evil is that our inheritance tree is....wiggly. And fixing that is a bigger job than can be attacked in this PR.